starter-project-web-vue2/store/samuel.js

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios'
+
+export const state = {
+    blogs: [],
+}
+export const actions = {
+    async fetchBlogs({ commit }) {
+        const response = await axios.get(
+            'https://blog-app-backend.onrender.com/api/articles/all'
+        )
+        commit('setBlogs', response.data)
+    },
+    async addBlog({ commit }, blogPost) {
+        console.log(blogPost, "store");
+        const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InNhbXVlbEBnbWFpbC5jb20iLCJzdWIiOiI2MmU2N2Q4ZGUwMDQ4MDdlNWUzMzQ1M2EiLCJpYXQiOjE2NTkyODUwNDIsImV4cCI6MTY1OTI4ODY0Mn0._KVpK16isY9xLk6bQRyVOMOLLEHwFy4jE2a7Ocsz-cU"
+        const response = await axios.post(
+            'https://blog-app-backend.onrender.com/api/articles',
+            blogPost, {
+                headers: {
+                    Authorization: 'Bearer ' + token,
+                }
+
+            }
+        )
+        commit('newBlog', response.data)
+    },
+    async deleteBlog({ commit }, id) {
+        console.log(id)
+        const token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InNhbXVlbEBnbWFpbC5jb20iLCJzdWIiOiI2MmU2N2Q4ZGUwMDQ4MDdlNWUzMzQ1M2EiLCJpYXQiOjE2NTkyODUwNDIsImV4cCI6MTY1OTI4ODY0Mn0._KVpK16isY9xLk6bQRyVOMOLLEHwFy4jE2a7Ocsz-cU"
```

Comment:
since you already implemented ur own login page. no need to add a token here. just use this.$axios() it would attach the token automatically if ure logged in